testscript/doc.go

```diff
@@ -120,6 +120,35 @@ when testing.Short() is false.
 
 Additional conditions can be added by passing a function to Params.Condition.
 
+An example:
+
+		Condition: func(cond string) (bool, error) {
+			// Assume condition name and args are separated by colon (":")
```

Comment:
I think that parsing the condition here is somewhat overkill tbh, and makes the example a bit longer than it needs to be. I'm pretty sure that people writing conditions are up to the task of extrapolating from something a bit simpler.

How about just something like:

```
Condition: func(cond string) (bool, error) {
    // Note: conditions can be more complex than this (for example [exists:/some/file]).
    if cond != "someCondition" {
        return false, fmt.Errorf("unknown condition")
    }
    // someCondition is a place-holder for any conditional logic you might want.
    return someCondition(), nil
}
```

?

Reply:
I can do that, if everyone agrees. However, I wanted the example to include some text manipulation logic, because I was baffled when I first tried to define a condition. What was unclear to me was that custom commands are defined as a map of functions, while custom conditions are just one function for all.
I think the current example will save users some grief.

Reply:
For what it's worth, I also feel this is likely to confuse people. But perhaps what this is really telling us is that the API should _be_ a map of condition names to `func(string) (bool, error)`. What do you think?